### PR TITLE
upgrade the default minimum tls version

### PR DESCRIPTION
**Pull request by YangKeao**
### What is changed, added or deleted? (Required)

Upgrade the default minimum tls version from TLS v1.1 to TLS v1.2

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/16185
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [x] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-configuration-file.md [hunk 1]
```diff
@@ -410,7 +410,7 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
 ### `tls-version`
 
 + 设置用于连接 MySQL 协议的最低 TLS 版本。
-+ 默认值：""，支持 TLSv1.1 及以上版本。
++ 默认值：""，支持 TLSv1.2 及以上版本。
```
Comment (shawn0915) [inline tidb-configuration-file.md:413]: 这里加个描述"<?版本支持 TLSv1.1及以上版本, >?版本支持 TLSv1.2 及以上版本" 是不是更好些？

**Review by qiancai — APPROVED** [review]

File: tidb-configuration-file.md (hunk 1 above)
Comment (qiancai) [inline tidb-configuration-file.md:413]:
```suggestion
+ 默认值：""，代表默认支持 TLSv1.2 及以上版本。在 v7.6.0 之前，TiDB 默认支持 TLSv1.1 及以上版本。
```

**Review by bb7133 — APPROVED** [review]
LGTM

**Comment by bb7133** [comment]
Please address the comments from @qiancai and @shawn0915

File: tidb-configuration-file.md (hunk 1 above)
Comment (YangKeao) [reply tidb-configuration-file.md:413]: 感谢提醒，确实会更好。

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**